cocos/core/animation/animation-component.ts

```diff
@@ -479,10 +491,6 @@ export class Animation extends Eventify(Component) {
     private _blendStateBuffer = new LegacyBlendStateBuffer();
 
     private _onAnimationSystemUpdate (deltaTime: number) {
-        if (!this._playing) {
```

Comment:
Doesn't feel right

Reply:
This behaviour matches that prior to https://github.com/cocos/cocos-engine/pull/10485